docusaurus.config.js

```diff
@@ -42,17 +42,105 @@ const config = {
+          },
+          {
+            from: "/eigenda-guides/eigenda-rollup-user-guides/",
+            to: "/eigenda/rollup-guides/tutorial",
+          },
+          {
+            from: "/eigenda-guides/eigenda-rollup-user-guides/building-on-top-of-eigenda",
+            to: "/eigenda/rollup-guides/tutorial",
+          },
+          {
+            from: "/eigenda-guides/eigenda-rollup-user-guides/system-performance-and-customization",
+            to: "/eigenda/system-performance-and-customization",
+          },
+          {
+            from: "/eigenda-guides/eigenda-rollup-user-guides/blob-explorer",
+            to: "/eigenda/blob-explorer",
+          },
+          {
+            from: "/eigenda-guides/eigenda-rollup-user-guides/op-stack-+-eigenda-user-guide",
+            to: "/eigenda/rollup-guides/op-stack",
+          },
+          {
+            from: "/eigenda-guides/eigenda-rollup-user-guides/op-stack-+-eigenda-user-guide/deploying-op-stack-+-eigenda-locally",
+            to: "/eigenda/rollup-guides/op-stack/deploying-op-stack-+-eigenda-locally",
+          },
+          {
+            from: "/eigenda-guides/integrations-overview",
+            to: "/eigenda/integrations-overview",
+          },
+          {
+            from: "/operator-guides/avs-installation-and-registration/eigenda-operator-guide/eigenda-avs-installation-registration-and-upgrade",
+            to: "/eigenda/operator-guides/eigenda-avs-installation-registration-and-upgrade",
+          },
+          {
+            from: "/operator-guides/avs-installation-and-registration/eigenda-operator-guide/",
+            to: "/eigenda/operator-guides/overview",
+          },
+          {
+            from: "/operator-guides/avs-installation-and-registration/eigenda-operator-guide/eigenda-metrics-and-monitoring",
+            to: "/eigenda/operator-guides/eigenda-metrics-and-monitoring",
+          },
+          {
+            from: "/operator-guides/avs-installation-and-registration/eigenda-operator-guide/ip-stability-and-costs-of-change",
+            to: "/eigenda/operator-guides/ip-stability-and-costs-of-change",
+          },
+          {
+            from: "/operator-guides/avs-installation-and-registration/eigenda-operator-guide/ejection-non-signing",
+            to: "/eigenda/operator-guides/ejection-non-signing",
+          },
+          {
+            from: "/operator-guides/avs-installation-and-registration/eigenda-operator-guide/troubleshooting",
+            to: "/eigenda/operator-guides/troubleshooting",
+          },
+          {
+            from: "/faqs/eigenda-operator-faq",
+            to: "/eigenda/operator-guides/operator-faq",
+          },
+          {
+            from: "/faqs/operator-general-faq",
+            to: "/eigenlayer/operator-guides/operator-faq",
           },
         ],
+        createRedirects(existingPath) {
+
+          // eigenlayer redirects
+          if (existingPath.includes('/eigenlayer')) {
```

Comment:
Hi @teddyknox I briefly tested the redirects against an existing URL (that does not include "eigenlayer" in the url). Can you confirm whether this returns a 404 for you or not? Thank you

http://localhost:3000/restaking-guides/restaking-user-guide/native-restaking/

Reply:
Are you using `yarn start` or `yarn build && yarn serve`? `yarn start` does not generate any redirects because it's only used for development. `yarn build && yarn serve` will allow you to test the redirects that would be there in production.

Reply:
You're right and that makes sense, thanks